sv/Vehicle.py

```diff
@@ -323,19 +323,23 @@ class PV(Vehicle):
     - agentacceleration (not implemented)
     - timetoacceleration (not implemented)
     """
-    def __init__(self, vid, name, start_state, frenet_state, yaw, path, debug_shdata, keep_active = True, length:float=VEHICLE_LENGTH, width:float=VEHICLE_WIDTH):
+    def __init__(self, vid, name, start_state, frenet_state, yaw, path, debug_shdata, scenario_vehicles, keep_active = True, length:float=VEHICLE_LENGTH, width:float=VEHICLE_WIDTH, collision_vid=None, set_speed=None, collision_point=None, use_collision_point=False):
         super().__init__(vid, name, start_state, frenet_state, yaw=yaw, length=length, width=width)
         self.type = Vehicle.PV_TYPE
         self.path = path
         self._debug_shdata = debug_shdata
+        self.collision_vid = collision_vid
```

Comment:
I added these fields to Actor and the values should be set by calling the super constructor on line 327